src/openforms/authentication/contrib/digid_oidc/models.py

```diff
@@ -0,0 +1,201 @@
+from django.conf import settings
+from django.db import models
+from django.utils.encoding import force_text
+from django.utils.translation import gettext_lazy as _
+
+import mozilla_django_oidc_db.settings as oidc_settings
+from django_better_admin_arrayfield.models.fields import ArrayField
+from solo.models import SingletonModel, get_cache
+
+from openforms.authentication.constants import AuthAttribute
+
+from .settings import CUSTOM_OIDC_DB_PREFIX
+
+
+def get_default_scopes():
+    """
+    Returns the default scopes to request for OpenID Connect logins
```

Comment:
it should probably be documented that these scopes are hardcoded and must be provided/supported by the IDP?